src/styles/primeng-theme/components/_multiselect.scss

```diff
@@ -0,0 +1,158 @@
+#multiselect-item {
```

Comment:
Where are you usign this?

Reply:
this is to align and set the margins for the items of  the multiselect

Reply:
Fixed. I was missing the @extend.